### PR TITLE
Updating DevPortal for upcoming release ARROWHEAD-6.4

### PR DESCRIPTION
**Pull request by rmoreliovlabs**
Adding support/context for the stableGasPrice feature in the reference page

### CODE REVIEW

**Comment by owans** [comment]
Thanks @rmoreliovlabs, is reproducible build available? We also need to update these sections too: 

- https://dev.rootstock.io/node-operators/setup/reproducible-build/
- https://dev.rootstock.io/developers/libraries/rsk-precompiled-abis/

File: docs/03-node-operators/04-setup/03-configuration/03-reference.md [hunk 1]
```diff
@@ -347,6 +348,57 @@ scoring {
 Check out [Configure RSKj node for mining](/node-operators/merged-mining/configure-mining)
 for detailed information about the `miner` configuration.
 
+## miner.stableGasPrice
+
+The `stableGasPrice` feature allows miners to set a predictable gas price for transactions, establishing a minimum rate that is less affected by market volatility and better aligned with fiat currency values like USD or EUR. This enhances cost management and improves overall transaction efficiency.
+
+The source configuration supports two methods: ETH_CALL and HTTP_GET.
```
Comment (owans) [inline docs/03-node-operators/04-setup/03-configuration/03-reference.md]:
```suggestion
The source configuration supports two methods: `ETH_CALL` and `HTTP_GET`.
```

File: docs/03-node-operators/04-setup/03-configuration/03-reference.md [hunk 2]
```diff
@@ -347,6 +348,57 @@ scoring {
...
+            data = "0xdf16e52d"
+        }
+    }
+
+    minStableGasPrice = 476190000000 # Minimum gas price
+    refreshRate = 6 hours          # How often to check for updates
+}
+```
+
+- **Source**: Gets the gas price from a specific smart contract
```
Comment (owans) [inline docs/03-node-operators/04-setup/03-configuration/03-reference.md]:
```suggestion
- **Source**: Retrieves the gas price from a specific smart contract
```

Comment (owans) [reply docs/03-node-operators/04-setup/03-configuration/03-reference.md]: Let's use either retrieve or get. See `eth_call` definition where you used retrieve

File: docs/03-node-operators/04-setup/03-configuration/03-reference.md [hunk 3]
```diff
@@ -347,6 +348,57 @@ scoring {
...
+    }
+
+    minStableGasPrice = 476190000000 # Minimum gas price
+    refreshRate = 6 hours          # How often to check for updates
+}
+```
+
 ## blockchain.config.name
 
 A string that describe the name of the configuration.
```
Comment (owans) [inline docs/03-node-operators/04-setup/03-configuration/03-reference.md]:
```suggestion
A string that describes the name of the configuration.
```

File: docs/03-node-operators/04-setup/03-configuration/03-reference.md [hunk 4]
```diff
@@ -347,6 +348,57 @@ scoring {
...
+            from = "0x0000000000000000000000000000000000000000"
+            to = "0xCA41630b09048b6f576459C34341cAB8Dc2cBE7E"
+            data = "0xdf16e52d"
+        }
+    }
+
+    minStableGasPrice = 476190000000 # Minimum gas price
+    refreshRate = 6 hours          # How often to check for updates
+}
+```
```
Comment (owans) [inline docs/03-node-operators/04-setup/03-configuration/03-reference.md:377]: The data inside params and after "ETH_CALL" doesn't need a comma?

Comment (rmoreliovlabs) [reply docs/03-node-operators/04-setup/03-configuration/03-reference.md:377]: Not exactly! The configuration I used when setting up the node for testing follows that format. It doesn't require a comma in that context

Comment (owans) [reply docs/03-node-operators/04-setup/03-configuration/03-reference.md:377]: Ok. Cool

File: docs/03-node-operators/04-setup/03-configuration/03-reference.md (hunk 3 above)
Comment (rmoreliovlabs) [reply docs/03-node-operators/04-setup/03-configuration/03-reference.md]: solved

File: docs/03-node-operators/04-setup/03-configuration/03-reference.md (hunk 2 above)
Comment (rmoreliovlabs) [reply docs/03-node-operators/04-setup/03-configuration/03-reference.md]: addressed

File: docs/03-node-operators/04-setup/03-configuration/03-reference.md (hunk 1 above)
Comment (rmoreliovlabs) [reply docs/03-node-operators/04-setup/03-configuration/03-reference.md]: solved

**Review by owans — APPROVED** [review]
LGTM!

**Pull request merged**